cdap-docs/developer-manual/source/cloud-runtimes/provisioners/gcp-dataproc.rst

```diff
@@ -8,5 +8,95 @@
 Google Dataproc
 ===============
 
-Describe what Dataproc is, then describe each config setting
+Dataproc is a Google Cloud Platform (GCP) service that manages Hadoop clusters in the cloud.
```

Comment:
Cloud Dataproc is a Google Cloud Platform (GCP) service that manages Hadoop clusters in the cloud and can be used to create large clusters quickly. The Google Dataproc provisioner simply calls the Cloud Dataproc APIs to create and delete clusters in your GCP account. The provisioner exposes several configuration settings that control what type of cluster is created.